### DISCUSSION TITLE
Creation of Devicon Project Board

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#536] [Ideas]
Hello guys!

In order to keep track of the tasks that need to be done between all maintainers, I've created [a project board](https://github.com/devicons/devicon/projects/4). These project boards are simplified Kanban/Trello boards so they are very easy to use.

I noticed that we have a bit of stuff in the discussions that were never taken on. I think formally putting them in the project board would be help for us to remember these items.

You don't have to use the board. However, if you do, please put your name on tasks that you are planning to do.

EDIT: tagging @amacado and @Panquesito7 since they are maintainers as well :)

**Comment by Panquesito7** [comment]
Sounds pretty well. Thanks! 😃

**Comment by amacado** [comment]
We can try this and see if it's usefull! :)
